site/content/blog/2020-12-01-whats-new-in-svelte-december-2020.md

```diff
@@ -11,7 +11,7 @@ It's the last "What's new in Svelte" of the year and there's lots to celebrate!
 
 1. `$$props`, `$$restProps`, and `$$slots` are all now supported in custom web components (**3.29.5**, [Example](https://svelte.dev/repl/ad8e6f39cd20403dacd1be84d71e498d?version=3.29.5)) and  `slot` components now support spread props: `<slot {...foo} />` (**3.30.0**)
 2. A new `hasContext` lifecycle function makes it easy to check whether a `key` has been set in the context of a parent component (**3.30.0** & **3.30.1**, [Docs](https://svelte.dev/docs#hasContext))
-3. `SvelteComponent` is now typed which makes it easier to add typed classes that extend base Svelte Components. Component library and framework authors rejoice! An example: `export class YourComponent extends SvelteComponent<{aProp: boolean}, {click: MouseEvent}, {default: {aSlot: string}}> {}` (**3.30.0**, [RFC](https://github.com/sveltejs/rfcs/pull/37))
+3. There is now a new `SvelteComponentTyped` class which makes it easier to add strongly typed components that extend base Svelte components. Component library and framework authors rejoice! An example: `export class YourComponent extends SvelteComponentTyped<{aProp: boolean}, {click: MouseEvent}, {default: {aSlot: string}}> {}` (**3.30.0**, [RFC](https://github.com/sveltejs/rfcs/pull/37))
```

Comment:
Was this broken to begin with? Otherwise this is a breaking change, no? The syntax `extends SvelteComponent<...> { ... }` ceases to work, right?

Reply:
Yes that syntax was introduced in `3.30` but reverted in `3.31` because it introduced a breaking change. `const a: typeof SvelteComponent = ASubclassOfSvelteComponent` threw errors which it did not before.